website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+
+## [`discussions-and-proposals`](https://github.com/react-native-community/discussions-and-proposals) 🗣
+
+This other ‘meta’ repository, created in July, expanded on the idea of a more open environment for conversations on React Native; while, partially, this need was covered by a few issues labelled [`For Discussion`](https://github.com/facebook/react-native/labels/For%20Discussion) in the main repository, we wanted to give our own spin to the RFC approach that other libraries (like React) have.
+
+This experiment found immediately its role in the React Native lifecycle: even the FB team is now relying heavily on it to discuss what could [be improved in React Native](https://github.com/react-native-community/discussions-and-proposals/issues/64), and coordinate the efforts around the [Lean Core project](https://github.com/react-native-community/discussions-and-proposals/issues/6) - among other interesting conversations.
+
+But we are far from done.
+
+### What awaits ahead 🎓
+
+As explained above, over the past 6-8 months, the [React Native Community organisation](https://github.com/react-native-community) has been used by core contributors to take more ownership over the development of React Native, and enhance the collaboration with Facebook. But this always lacked the formal structure that similar projects may have in place. 
+
+By enforcing a set of standards for all the packages/repos hosted in it, providing a common place for all the maintainers to help each other and providing quality code, this organisation can set the example for everyone in the larger developer community.
+
+We are planning, for the early 2019, to have this new set of guidelines in place - you can let us know what you think in the [dedicated discussion](https://github.com/react-native-community/discussions-and-proposals/issues/63), and in the manifesto proposal that will be written soon.
+
+With this last founding stone, we are confident that the community will become more inclusive and when 1.0 will be reached, we will all be able to write (even more) awesome apps, capable of leveraging this joined effort 🤗
+
+----
+
+I hope you are as excited as I am in looking at the future of this community, and that I will see all of you involved either in the conversations happening in the repositories listed above - or via the awesome code you’ll produce.
+
+We are aware that our approach to communicate these efforts has not been as effective as we liked, and in an attempt to give you all an easier time keeping up with everything going on in the React Native Community (from releases to active discussions) we created a new twitter account that you can rely on [@ReactNativeComm](https://twitter.com/ReactNativeComm) (if you are not on it remember that you can always watch repositories via GitHub). It will become more actively starting 2019, but feel free to start following it right away 🐣
```

Comment:
This seems added in as an afterthought, the previous paragraph seems to be the closing paragraph. May consider a better place to add this in

Reply:
I see what you are saying, but tbh I think it's ok to put it at the end because it's basically "Off Topic" and here it's the 'least bad' spot to have this communicated 😅

Reply:
What if it was one of the three header points, added to line 32:

```

## [@ReactNativeComm](https://twitter.com/ReactNativeComm)`

<move this paragraph here>
```

That seems inline with the lead-in sentence for these points: 

> These members took a few substantial steps towards making the whole community more empowered to shape the future of this project. _They taken these steps with this goal in mind_[edited]:

And this would promote it to "on topic" :D

Reply:
that's a good idea! I'll edit